### PR TITLE
[DO NOT MERGE] Try markus suggestion

### PR DESCRIPTION
**Pull request by aliok**
Trying @markusthoemmes suggestion (https://github.com/openshift-knative/serverless-operator/pull/570#pullrequestreview-509511186) here since I don't want to interrupt ci in https://github.com/openshift-knative/serverless-operator/pull/570

### CODE REVIEW

File: test/lib.bash [hunk 1]
```diff
@@ -92,6 +93,9 @@ function downstream_serving_e2e_tests {
   return $failed
 }
 
+function downstream_knative_kafka_e2e_tests {
+}
```
Comment (markusthoemmes) [inline test/lib.bash:97]: Lacks the fix for this I fear, so I guess it'll stumble before even trying to compile? :thinking:

Comment (aliok) [reply test/lib.bash:97]: aw man, crap!

Comment (markusthoemmes) [reply test/lib.bash:97]: Too many balls in the air? :juggling_person:

Comment (aliok) [reply test/lib.bash]: thanks. rebased now!

**Comment by aliok** [comment]
/test 4.5-upgrade-tests-aws-ocp-45

AWS provision problem

**Comment by matzew** [comment]
because of #581 
/close

**Pull request closed without merging**